spec/lib/authorities/linked_data/term_config_spec.rb

```diff
@@ -266,7 +280,7 @@
     it 'returns nil if only search configuration is defined' do
       expect(search_only_config.term_results_altlabel_ldpath).to eq nil
     end
-    it 'return nil if altlabel predicate is not defined' do
+    it 'return nil if altlabel ldpath is not defined' do
```

Comment:
The changes from comment saying predicate to ldpath is just fixing copy/paste/fail to update errors in a previous PR.